lib/http.js

```diff
@@ -31,7 +31,7 @@ class HTTPClient {
 
     async getSnapshot () {
         try {
-            return await this.client.get('live/snapshot').json()
+            return await this.client.get('live/snapshot', { timeout: { request: 20000 } }).json()
```

Comment:
In the absence of any user settable property (where they can manage the timeout value themselves), is there any reason we dont crank this a little further? like 30s or 45s ?

Reply:
I didn't want to encourage even bigger flows... but we could go to 30 i suppose.

Reply:
```suggestion
            return await this.client.get('live/snapshot', { timeout: { request: 30000 } }).json()
```

I like 30 :)